sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -33,3 +33,27 @@ apply from: "../common.gradle"
 
 // TODO(BEAM-8954): Remove this once tox uses isolated builds.
 testPy38Cython.mustRunAfter testPython38, testPy38CloudCoverage
+
+toxTask "whitespacelint", "whitespacelint"
+
+task archiveFilesToLint(type: Zip) {
+  archiveFileName = "files-to-whitespacelint.zip"
+  destinationDirectory = file("$buildDir/dist")
+
+  from ("$rootProject.projectDir") {
+    include "**/*.md"
+    include "**/build.gradle"
+  }
+}
+
+task unpackFilesToLint(type: Copy) {
+  from zipTree("$buildDir/dist/files-to-whitespacelint.zip")
+  into "$buildDir/files-to-whitespacelint"
+}
+
+whitespacelint.dependsOn archiveFilesToLint, unpackFilesToLint
+unpackFilesToLint.dependsOn archiveFilesToLint
+archiveFilesToLint.dependsOn cleanPython
+archiveFilesToLint.mustRunAfter cleanPython
+unpackFilesToLint.mustRunAfter cleanPython, archiveFilesToLint
```

Comment:
did you find that you needed the `mustRunAfter` rules even though you also had the `dependsOn` rules?

Reply:
Removed the mustRunAfter statements.